extensions/machine-learning/src/views/models/prediction/columnsTable.ts

```diff
@@ -367,22 +384,58 @@ export class ColumnsTable extends ModelViewBase implements IDataComponent<Predic
 		return [];
 	}
 
-	private createWarningButton(message: string): azdata.ButtonComponent {
+	private createWarningButton(): azdata.ButtonComponent {
 		const warningButton = this._modelBuilder.button().withProperties({
-			width: '16px',
-			height: '16px',
-			title: message,
+			width: `${_warningButtonDimensions.width}px`,
+			height: `${_warningButtonDimensions.height}px`,
+			title: constants.columnDataTypeMismatchWarningHelper,
 			iconPath: {
 				dark: this.asAbsolutePath('images/warning.svg'),
 				light: this.asAbsolutePath('images/warning.svg'),
 			},
-			iconHeight: '16px',
-			iconWidth: '16px'
+			iconHeight: `${_warningButtonDimensions.height}px`,
+			iconWidth: `${_warningButtonDimensions.width}px`
 		}).component();
 
 		return warningButton;
 	}
 
+	public openWarningCalloutDialog(dialogHeading: string, dialogName?: string, calloutMessageText?: string, calloutMessageLinkText?: string, calloutMessageLinkUrl?: string, dialogProperties?: azdata.window.IDialogProperties): void {
+		let dialog = azdata.window.createModelViewDialog(dialogHeading, dialogName, 'narrow', 'calloutCompact', 'left', true, false, dialogProperties);
+		let warningTab: azdata.window.DialogTab = azdata.window.createTab('warning');
+		warningTab.registerContent(async view => {
+			let warningContentContainer = view.modelBuilder.divContainer().withProperties({
```

Comment:
general feedback, if you don't need to change the variable, you should use `const` instead of `let`